public/data/AntonyKimani.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "ANTONY KIMANI",
```

Comment:
Is it needed to capitalise the whole name? I think now, but I am not sure what others would say. @AntonyKimani 
Thank you.

Reply:
Thanks @CodingSpecies  For pointing that out .